test.feature

```diff
@@ -0,0 +1,7 @@
+Feature:
```

Comment:
This feature file at the root looks like a debugging file to me, that should not be committed.